src/main/java/de/flo56958/minetinker/listeners/EasyHarvestListener.java

```diff
@@ -77,15 +78,15 @@ private static void breakWarts(Player p, ItemStack tool, Block b) {
                     Block b2;
 
                     if (PlayerInfo.getFacingDirection(p).equals("N") || PlayerInfo.getFacingDirection(p).equals("S")) {
-                        if (config.getBoolean("Modifiers.Power.lv1_vertical")) {
```

Comment:
Mistake from me, but this config option does not exist anymore. It is now in the Power configuration. The out come will be always false here.

Reply:
When I refactored package names it seems a lot of things were accidentally changed along with it.
This might be partially why things are extremely broken.

I'll change these to use the Power config at some point.

Reply:
For now I'm going to leave them as is and try to replicate the pre-implementation state of the plugin and fix the issues caused by the refactor.

Reply:
Refactoring Package names and strings in one are usually not really good for the code. (Did that myself at one point with the plugin)

Reply:
I didn't intend to at the time but intellij decided to do it anyway. I reverted the changes and fixed a few bugs and everything seems to be working normally.